app/views/ats/placements/_placement_pipeline_card.html.slim

```diff
@@ -8,7 +8,7 @@
 
   .d-flex.gap-3
     .card-avatar.col-auto
-      = picture_avatar_icon placement.candidate.avatar, {}, class: "w-100 h-100"
+      = picture_avatar_icon placement.candidate.avatar, {}, class: "w-100 h-100 avatar-64px"
```

Comment:
Yes, the avatar will have 2 classes, `avatar-sm` and `avatar-64px`, but only the last one will work.
We probably need to unify the method for displaying avatars (maybe even create a component for that?).

Reply:
Component sounds good.